.test-defs/CreateShootForSeed.yaml

```diff
@@ -0,0 +1,42 @@
+kind: TestDefinition
+metadata:
+  name: create-shoot-for-seed
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the creation of a shoot.
+  activeDeadlineSeconds: 7200
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/system/shoot_creation
```

Comment:
the same test is called as in https://github.com/gardener/gardener/blob/master/.test-defs/CreateShoot.yaml why do we need a second testdef?